daemon/cmd/daemon_main.go

```diff
@@ -292,6 +292,7 @@ func initializeFlags() {
 
 	flags.StringP(option.IpvlanMasterDevice, "", "undefined", "Device facing external network acting as ipvlan master")
 	option.BindEnv(option.IpvlanMasterDevice)
+	flags.MarkDeprecated(option.IpvlanMasterDevice, "This option will be removed in v1.12")
```

Comment:
Command reference is generated by running `cilium-agent --cmdref`. This does not output deprecated arguments, so now that this is documented (and I found the `make -C Documentation update-cmdref`), it will no longer show up there. This is unrelated to the helm charts under `install/kubernetes/`.

I've already updated `install/kubernetes/cilium/values.yaml` for the IPVLAN features to ensure they are marked as deprecated in the helm docs.

Was there somewhere else that's missing the relevant notifications?

Reply:
Apologies, OK in that case we're good.